mkdocs.yml

```diff
@@ -59,6 +59,7 @@ pages:
   - 'Web Client': 'faq/web-client.md'
   - 'Blocks': 'faq/blocks.md'
   - Proof-of-stake:
+    - 'General': 'faq/proof-of-stake/general.md'
```

Comment:
You need to add this to the other language yml files as well (in english of course).